coreMacros/src/main/scala/chisel3/internal/sourceinfo/SourceInfoTransform.scala

```diff
@@ -27,14 +27,6 @@ trait SourceInfoTransformMacro {
   def implicitCompileOptions = q"implicitly[_root_.chisel3.core.CompileOptions]"
 }
 
-class WireTransform(val c: Context) extends SourceInfoTransformMacro {
-  import c.universe._
-  def apply[T: c.WeakTypeTag](t: c.Tree): c.Tree = {
-    val tpe = weakTypeOf[T]
-    q"$thisObj.do_apply($t, null.asInstanceOf[$tpe])($implicitSourceInfo, $implicitCompileOptions)"
-  }
-}
```

Comment:
They're unnecessary, the main reason we needed transforms is so you can chain an apply. But if you chain an apply after a Reg / Wire call, you're basically discarding bits, which seems like an odd use case.